### DISCUSSION TITLE
source code for screen UI?

### DISCUSSION THREAD

**Discussion by ghost** [NARUTOfzr/Neptune_3#35] [Q&A]
can we please have the source code for the UI so we can change it to our liking

**Comment by NARUTOfzr** [comment]


**Reply by ghost** [reply]
Thanks man I appreciate it

**Comment by TheFeralEngineer** [comment]
I got this from Johnson a week ago and the version I received from him gives me the same open error that this one does. Is there a way to resolve this issue? It says Open in error, please import config file or restoring

**Reply by NARUTOfzr** [reply]
> I got this from Johnson a week ago and the version I received from him gives me the same open error that this one does. Is there a way to resolve this issue? It says Open in error, please import config file or restoring
@TheFeralEngineer   I think it might be a problem with the computer development environment. Because the company computer can open the file normally, but the home computer will encounter the same problem.
I don't have a solution yet.
@purplewazard Do you have the same problem?
![企业微信截图_16677221166345](https://user-images.githubusercontent.com/64059773/200506203-3e15a002-fa6b-49df-81f9-5041aac77be0.png)

**Reply by ghost** [reply]
yes i do i get the same thing but i can see the start up amation frames

**Reply by TheFeralEngineer** [reply]
has there been any progress with this? i have a makerbase h43 project that seems to work perfectly in dgus and this doesn't

**Reply by TheFeralEngineer** [reply]
Gloria from dwin can open this on her computer, but I cannot 🥺

**Comment by TheFeralEngineer** [comment]
Same here. I asked Gloria at Dwin for DGUS technical support

Phil T.
The Feral Engineer

Check out my LinuxCNC tutorials, machine builds and other antics at
www.youtube.com/c/theferalengineer

Help support my channel efforts and coffee addiction:
www.patreon.com/theferalengineer

Order one of the coolest label makers on the market at
http://labelworks.epson.com, use coupon code "theferalengineer" and receive
20% off of your order 🙂

**Comment by NARUTOfzr** [comment]
@purplewazard @TheFeralEngineer @purplewazard 
The road was rough, but there were still a few clues.
[DGUS_Q.zip](https://github.com/NARUTOfzr/Neptune_3/files/10095512/DGUS_Q.zip)

**Reply by TheFeralEngineer** [reply]
That seems to work! Thank you.

**Reply by ghost** [reply]
yipeeeee

**Reply by ghost** [reply]
@NARUTOfzr @TheFeralEngineer i opened the hmi file i get a microsoft .net framwork error

![image](https://user-images.githubusercontent.com/109108137/204879438-94653cce-dc85-4b84-9d73-7aedde072b03.png)
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.ArgumentException: The UNC path should be of the form \\server\share.
   at System.IO.Path.NormalizePathFast(String path, Boolean fullCheck)
   at System.IO.Path.NormalizePath(String path, Boolean fullCheck)
   at System.IO.Path.GetFullPathInternal(String path)
   at System.IO.DirectoryInfo..ctor(String path)
   at BizDraw.Controls.GetFilelist.FileList(String filePath, String Ext)
   at BizDraw.Globel.InitMatfonts()
   at BizDrawClient.Form1.打开OToolStripButton_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.Label.WndProc(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   at System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)


************** Loaded Assemblies **************
mscorlib
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.9164 (WinRelRS6.050727-9100)
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework/v2.0.50727/mscorlib.dll
----------------------------------------
DGUS_V7.642
    Assembly Version: 7.6.4.2
    Win32 Version: 7.6.4.2
    CodeBase: file:///C:/pretty%20much%20all%20my%20files/3d%20printer/neptune%203/screen%20firmware/DGUS_V7642_20220922/DGUS_V7642_20220922/DGUS_V7642/DGUS_V7.642.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.9157 (WinRelRS6.050727-9100)
    CodeBase: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Windows.Forms/2.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.9157 (WinRelRS6.050727-9100)
    CodeBase: file:///C:/WINDOWS/assembly/GAC_MSIL/System/2.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.9157 (WinRelRS6.050727-9100)
    CodeBase: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Drawing/2.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
DwinTerminal
    Assembly Version: 7.3.6.0
    Win32 Version: 7.3.6.0
    CodeBase: file:///C:/pretty%20much%20all%20my%20files/3d%20printer/neptune%203/screen%20firmware/DGUS_V7642_20220922/DGUS_V7642_20220922/DGUS_V7642/DwinTerminal.DLL
----------------------------------------
DWINUI
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.0.0
    CodeBase: file:///C:/pretty%20much%20all%20my%20files/3d%20printer/neptune%203/screen%20firmware/DGUS_V7642_20220922/DGUS_V7642_20220922/DGUS_V7642/DWINUI.DLL
----------------------------------------
System.Xml
    Assembly Version: 2.0.0.0
    Win32 Version: 2.0.50727.9157 (WinRelRS6.050727-9100)
    CodeBase: file:///C:/WINDOWS/assembly/GAC_MSIL/System.Xml/2.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.
